formats/definitions.json

```diff
@@ -550,6 +550,67 @@
           }
         }
       }
+    },
+    "hmrc_manual_change_notes": {
+      "description": "A history of changes to HMRC manuals and the associated section. section_id is included and required making it different to manual_change_notes",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "additionalProperties": false,
+        "required": [
+          "section_id",
```

Comment:
We could merge these definitions as although section_id is "required" for hmrc manuals change notes - it's not actually used by manuals-frontend when rendering, so a single definition with optional section_id would work.